packages/opentelemetry-node/docs/get-started.md

```diff
@@ -0,0 +1,135 @@
+<!--
+Goal of this doc:
+The user is able to successfully see data from their Node application make it to the Elastic UI via the Elastic Distribution for OpenTelemetry Node.js
+
+Assumptions we're comfortable making about the reader:
+* They are familiar with Elastic
+* They are sending data to Elastic
+* They have Node and NPM installed
+-->
+
+# Get started
+
+> [!WARNING]
+>  The Elastic Distribution for OpenTelemetry Node.js is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features.
+>
+> We welcome your feedback! You can reach us by [opening a GitHub issue](https://github.com/elastic/elastic-otel-node/issues) or starting a discussion thread on the [Elastic Discuss forum](https://discuss.elastic.co/tags/c/observability/apm/58/nodejs).
+
+This guide shows you how to use the Elastic Distribution for OpenTelemetry Node.js ("the distro")
+to instrument your Node.js application and send OpenTelemetry data to an Elastic Observability deployment.
+
+This doc will guide you through the minimal configuration options to get the Elastic distro set up in your application.
+You do _not_ need any existing experience with OpenTelemetry to set up the Elastic distro initially.
+If you need more control over your configuration after getting set up, you can learn more in [OpenTelemetry SDK documentation](https://opentelemetry.io/docs/languages/js).
```

Comment:
I'm getting some mixed messages about how we should be framing this. My response to these mixed messages in the Java docs was to replace this paragraph with this:

>**Already familiar with OpenTelemetry?** It's an explicit goal of this distribution to introduce _no new concepts_ outside those defined by the wider OpenTelemetry community.
>
>**New to OpenTelemetry?** This section will guide you through the _minimal_ configuration options to get the Elastic distro set up in your application. You do _not_ need any existing experience with OpenTelemetry to set up the Elastic distro initially. If you need more control over your configuration after getting set up, you can learn more in the [OpenTelemetry documentation](https://opentelemetry.io/docs/zero-code/java/agent/).

Would it make sense to do something similar here?

Reply:
Yes, that looks good.